docker-compose.yml

```diff
@@ -0,0 +1,31 @@
+version: "3"
```

Comment:
I needed a way to quickly validate the changes to the docker file, so since now we need PgSQL I might as well create a docker-compose file while I'm at it.